engine/battle/core.asm

```diff
@@ -5201,7 +5202,8 @@ MoveSelectionScreen:
 	pop hl
 	call BattleMoveDescTextbox
 	call WaitPressAorB_BlinkCursor
-	jr .start_over
+	call LoadWeatherIconSprite
+	jp .start_over
```

Comment:
```suggestion
	jmp .start_over
```
Polished Crystal uses `jmp` instead of `jp`. `jmp` is a macro that evaluates to a normal `jp` except that it also comes with an assertion that will warn during build time if it ever can be optimized back to being a `jr`.